examples/elastic/src/main/java/com/hazelcast/jet/examples/elastic/ElasticSourceExample.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.hazelcast.jet.examples.elastic;
+
+import com.hazelcast.jet.Jet;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.Observable;
+import com.hazelcast.jet.datamodel.Tuple2;
+import com.hazelcast.jet.elastic.ElasticSources;
+import com.hazelcast.jet.pipeline.BatchSource;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static com.hazelcast.jet.aggregate.AggregateOperations.mapping;
+import static com.hazelcast.jet.aggregate.AggregateOperations.toList;
+import static com.hazelcast.jet.datamodel.Tuple2.tuple2;
+import static com.hazelcast.jet.pipeline.Pipeline.create;
+
+public class ElasticSourceExample {
```

Comment:
I suggest to prepare (and document) elastic code samples in form:
1. run empty elasticsearch
2. run ElasticSinkExample - it will store data to elastic
3. run ElasticSourceExample - it will read previously provided data from elastic and show them to user (through Observable or Sinks.logger)

Reply:
Yeah, good idea. this is how I run the demo. Do you mean somewhere in the example module (as readme) or as how to guide on the website?

Reply:
Originally I thought it should be in javadoc for both example classes but it can be added to readme or website as well.